client/containers/Pub/PubSubmissionHeader/spubHeader.scss

```diff
@@ -0,0 +1,72 @@
+@import 'styles/variables.scss';
+@import './themes.scss';
+
+$header-margin-top: 0px;
+$header-margin-bottom: 26px;
+$header-margin-bottom-mobile: 13px;
+$header-margin-total: $header-margin-top + $header-margin-bottom;
+$header-margin-total-mobile: $header-margin-top + $header-margin-bottom-mobile;
+$title-elements-spacing: 15px;
+$mobile-bottom-buttons-padding-top: 15px;
+$mobile-bottom-buttons-spacing: 10px;
+
+.pub-header-component {
```

Comment:
It's important (and non-obvious) that when you `require('./spubHeader.scss')` you're loading those styles _globally_, for the whole application, so it's not just for cosmetics that this class needs to have a unique name.

Looking at the difference between these styles and those in the existing `pubHeader.scss`, it looks like this is a copy-paste of that file with some stuff taken out. I'm tempted to say that this is fine for MVP-mode development, but I also wonder how far you'd get by factoring the shared styles into a `@mixin pub-header-basic` that's then `@include`-ed from both files.